File: 1502 can-make-arithmetic-progression-from-sequence/solution.cs
public class Solution {
    public bool CanMakeArithmeticProgression(int[] arr) {
        if(arr.Length<=2) return true;
        Array.Sort(arr);
        int gap=arr[1]-arr[0];
        for(int i=2;i<arr.Length;i++) {
            if(arr[i]-arr[i-1]!=gap) return false;
        }
        return true;
    }
}

File: 1502 can-make-arithmetic-progression-from-sequence/solution.rs
impl Solution {
    pub fn can_make_arithmetic_progression(mut arr: Vec<i32>) -> bool {
        if arr.len() <=2 {
            return true
        }
        arr.sort();
        let gap =arr[1]-arr[0];
        for i in 2..arr.len() {
            if arr[i]-arr[i-1]!=gap {
                return false
            }
        }
        true
    }
}
